src/pages/AppList.svelte

```diff
@@ -19,44 +23,159 @@
     filteredApps = apps.filter((appItem) => appItem.name.includes(filterText));
   }
   let showDeprecated = false;
+
+  const appLogos = {
+    browser: "/img/app-logos/browser.png",
+    beta: "/img/app-logos/beta.png",
+    amazon: "/img/app-logos/amazon.png",
+    "klar-focus": "/img/app-logos/klar-focus.png",
+    lockwise: "/img/app-logos/lockwise.png",
+    nightly: "/img/app-logos/nightly.png",
+    reality: "/img/app-logos/reality.png",
+    others: "/img/app-logos/mozilla.png",
+    dev: "/img/app-logos/dev.png",
+  };
+
+  function getAppLogo(app) {
+    if (app.match(/fenix/) || app.match(/nightly/)) {
+      return appLogos.nightly;
+    }
+    if (app.match(/beta/)) {
+      return appLogos.beta;
+    }
+    if (app.match(/echo/) || app.match(/fire-tv/)) {
+      return appLogos.amazon;
+    }
+    if (app.match(/focus/) || app.match(/klar/)) {
+      return appLogos["klar-focus"];
+    }
+    if (app.match(/lockwise/)) {
+      return appLogos.lockwise;
+    }
+    if (app.match(/reality/)) {
+      return appLogos.reality;
+    }
+    if (app.match(/dev/)) {
+      return appLogos.dev;
+    }
+    if (app.match(/firefox/)) {
+      return appLogos.browser;
+    }
+    return appLogos.others;
+  }
+
+  function isPlatform(app) {
+    if (
+      includes(app, "iOS") ||
+      includes(app, "Android") ||
+      includes(app, "Amazon")
+    ) {
+      return true;
+    }
+    return false;
+  }
+
+  function getPlatformLogo(app) {
+    if (includes(app, "iOS")) {
+      return "https://i.imgur.com/swlzpbM.jpg";
+    }
+    if (includes(app, "Android")) {
+      return "https://i.imgur.com/ji8NP0b.png";
+    }
+    if (includes(app, "Amazon")) {
+      return "https://i.imgur.com/EwaktM3.png";
+    }
+    return null;
```

Comment:
`undefined` is a bit more idiomatic in JS-land.

```suggestion
    return undefined;
```